tests/acceptance/acceptancesuites/acceptancesuite.js

```diff
@@ -46,84 +51,120 @@ test('Basic universal flow', async t => {
 });
 
 fixture`Vertical search page works as expected`
+  .beforeEach(async t => {
+    await VerticalPage.getVerticalResultsComponent().registerLogger(t);
+  })
   .before(setupServer)
   .after(shutdownServer)
   .page`${VERTICAL_PAGE}`;
 
 test('pagination flow', async t => {
   const searchComponent = VerticalPage.getSearchComponent();
+  const resultComponent = VerticalPage.getVerticalResultsComponent();
+
   await searchComponent.enterQuery('Virginia');
   await searchComponent.submitQuery();
+  await resultComponent.waitOnSearchComplete();
+
   const paginationComponent = VerticalPage.getPaginationComponent();
   await paginationComponent.clickNextButton();
+  await resultComponent.waitOnSearchComplete();
+
   const pageNum = await paginationComponent.getActivePageLabelAndNumber();
   await t.expect(pageNum).eql('Page 2');
 });
 
 test('navigating and refreshing mantains that page number', async t => {
+  const resultComponent = VerticalPage.getVerticalResultsComponent();
+
   await t.navigateTo(`${VERTICAL_PAGE}?query=Virginia`);
+  await resultComponent.waitOnSearchComplete();
 
   const paginationComponent = VerticalPage.getPaginationComponent();
   await paginationComponent.clickNextButton();
+  await resultComponent.waitOnSearchComplete();
+
   await browserRefreshPage();
+  await resultComponent.waitOnSearchComplete();
+
   const pageNum = await paginationComponent.getActivePageLabelAndNumber();
   await t.expect(pageNum).eql('Page 2');
 });
 
 test('navigating and refreshing mantains that page number with blank query', async t => {
   const searchComponent = VerticalPage.getSearchComponent();
+  const resultComponent = VerticalPage.getVerticalResultsComponent();
+
   await searchComponent.submitQuery();
+  await resultComponent.waitOnSearchComplete();
 
   const paginationComponent = VerticalPage.getPaginationComponent();
   await paginationComponent.clickNextButton();
+  await resultComponent.waitOnSearchComplete();
+
   let pageNum = await paginationComponent.getActivePageLabelAndNumber();
   await t.expect(pageNum).eql('Page 2');
+
   await browserRefreshPage();
+  await resultComponent.waitOnSearchComplete();
   pageNum = await paginationComponent.getActivePageLabelAndNumber();
   await t.expect(pageNum).eql('Page 2');
 });
 
 test('spell check flow', async t => {
+  const searchQueryUrl = VerticalPage.getVerticalResultsComponent().getSearchQueryUrl();
```

Comment:
I think the url should go into some kind of constants file instead of on the VerticalResultsComponent. either one of the existing utils files or a new one